pandas/tests/resample/test_resample_api.py

```diff
@@ -611,3 +611,76 @@ def test_resample_agg_readonly():
 
     result = rs.agg("min")
     tm.assert_series_equal(result, expected)
+
+
+def test_end_origin():
+
+    start, end = "2000-10-01 23:30:00", "2000-10-02 00:26:00"
+    rng = date_range(start, end, freq="7min")
+    ts = Series(np.arange(len(rng)) * 3, index=rng)
+
+    res = ts.resample("17min", origin="end").sum().astype("int64")
+    data = [0, 18, 27, 63]
```

Comment:
i'd stick that in the constructor

Reply:
mroeschke said constructing `ts` in each test was better.